executors/cmdexec/cmdexec.go

```diff
@@ -23,10 +23,14 @@ import (
 var (
 	logger     = logging.NewLogger("[cmdexec]", logging.LEVEL_DEBUG)
 	preallocRe = regexp.MustCompile("^[a-zA-Z0-9-_]+$")
+
+	// escape priviledges to host
+	HostExec = "exec-on-host"
 )
 
 type RemoteCommandTransport interface {
 	ExecCommands(host string, commands rex.Cmds, timeoutMinutes int) (rex.Results, error)
+	ExecHostCommands(host string, commands rex.Cmds, timeoutMinutes int) (rex.Results, error)
```

Comment:
I think this is the wrong layer for this. ExecCommands runs a command on a node and it shouldn't care if that command "breaks out" of a container or not. In fact, this seems like the wrong place to decide if the command is to be run on something "containerized" or not.